src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -234,7 +234,7 @@ private void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResults, Immu
 
                 if (!expected.HasLocation)
                 {
-                    verifier.Equal(Location.None, actual.Location, $"Expected:\nA project diagnostic with No location\nActual:\n{FormatDiagnostics(analyzers, actual)}");
+                    verifier.Equal(Location.None, actual.Location, $"Expected:\r\nA project diagnostic with No location\r\nActual:\r\n{FormatDiagnostics(analyzers, actual)}");
```

Comment:
Shouldn't we use `Environment.NewLine` here?

Reply:
Seems like yes, but this change matches the rest of the existing code. 😕 

I'll file a follow-up issue since it touches a bunch of other code.

Reply:
➡️ #287